### ISSUE TITLE
Last.fm's get_bio_summary() doesn't format nicely when there are multiple bands named the same

### ISSUE THREAD

**Issue by martijnboers** [martijnboers/BlotterTrax#20]
Some examples:

- Cut of at a weird position, actually two bands with this name https://www.reddit.com/r/listentothis/comments/f6hrys/vaz_visiting_hours_noise_rock_2013/fi4t8ml/?context=3

- Same here https://www.reddit.com/r/listentothis/comments/f6hrag/the_busy_signals_look_the_other_way_punkpowerpop/fi4t4co?utm_source=share&utm_medium=web2x

- Only shows one artist https://www.reddit.com/r/listentothis/comments/f6dot5/aviators_ghosts_of_our_fathers_dark_alternative/fi4s5ay?utm_source=share&utm_medium=web2x

- Same here https://www.reddit.com/r/listentothis/comments/f6bqmf/eggplant_pity_alternative1989/

Don't know what the best approach is, maybe the API exposes when an artist name has multiple records and it can be formatted differently

**Comment by Nedlinin** [comment]
Not noticing an easy way in LastFM to detect another artist of the same name and have found numerous other people complaining about this drawback of LastFM..

I was even looking at the possibility of getting the track based on artist name + song name and working back to the artist from there but even then the "artist" ends up being the merged page that could discuss multiple artists.

**Comment by Nedlinin** [comment]
Using one of the above posts as a test I was able to hit the lastfm api and get the follow:

`{
  "track": {
    "name": "Look the Other Way",
    "mbid": "4f3893ba-8065-4b64-8a74-1a30d511f18f",
    "url": "https://www.last.fm/music/The+Busy+Signals/_/Look+the+Other+Way",
    "duration": "107000",
    "streamable": {
      "#text": "0",
      "fulltrack": "0"
    },
    "listeners": "1299",
    "playcount": "5303",
    "artist": {
      "name": "The Busy Signals",
      "mbid": "93f2ceaa-4a70-4dbb-b0b8-617a50611353",
      "url": "https://www.last.fm/music/The+Busy+Signals"
    },
    "album": {
      "artist": "The Busy Signals",
      "title": "the Busy Signals",
      "mbid": "c4056e7d-5d7f-4d83-9303-19fa3d02081f",
      "url": "https://www.last.fm/music/The+Busy+Signals/the+Busy+Signals",
      "image": [
        {
          "#text": "https://lastfm.freetls.fastly.net/i/u/34s/16bb93073b5a40ad84ea7552d5377181.png",
          "size": "small"
        },
        {
          "#text": "https://lastfm.freetls.fastly.net/i/u/64s/16bb93073b5a40ad84ea7552d5377181.png",
          "size": "medium"
        },
        {
          "#text": "https://lastfm.freetls.fastly.net/i/u/174s/16bb93073b5a40ad84ea7552d5377181.png",
          "size": "large"
        },
        {
          "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/16bb93073b5a40ad84ea7552d5377181.png",
          "size": "extralarge"
        }
      ],
      "@attr": {
        "position": "2"
      }
    },
    "toptags": {
      "tag": []
    }
  }
}`

Might want to JSON pretty print ^

Long story short, the artist link back is still the merged artist but we can get the album information.. Might be worth a discussion with the mod team about coming up with rules based on an album playcount rather than total artist scrobbles.  

It also requires more API requests to retrieve as we first have to get the track info *then* the album from the information returned.

**Comment by TcMaX** [comment]
Regular Last.FM user here. Last.FM does not differentiate between multiple bands of the same name whatsoever. For instance, when I scrobble BRATS, a japanese rock band, the scrobble is tagged as danish, because there used to be another band called Brats. Nothing I can do about it.

I believe the only way to format the summary will be to write code to detect whether there are multiple bands based on the summary text, then 
get the full bio instead of the summary, and either print the full bio or create a summary from your own code

**Comment by Nedlinin** [comment]
Definitely disappointing to hear as that is most definitely not a trivial task!

**Comment by martijnboers** [comment]
Maybe we can look into using Discogs?

**Comment by martijnboers** [comment]
Will do some investigation on this tonight, see this as blocking beta release

**Comment by TcMaX** [comment]
> Maybe we can look into using Discogs?

Checked out the api a little bit, looks doable
